_posts/admin/2020-04-17-enable-mptv2.md

```diff
@@ -0,0 +1,24 @@
+---
+date: 2020-04-17
+title: Enabling Managed Pipeline Templates
+categories:
+   - Admin
+description: How to enable the Managed Pipeline Templates feature in Armory Spinnaker
+type: Document
+---
+
+Armory Spinnaker 2.19 contains the latest version of Managed Pipeline Templates, which is the default pipeline templating solution offered in OSS Spinnaker. This feature is disabled by default in Armory Spinnaker.
```

Comment:
The UI for this feature is disabled by default in Armory Spinnaker

Again @fernando-armory to confirm its just the UI thats being disabled, should still work through spin-cli